lib/util/git/index.ts

```diff
@@ -418,9 +418,11 @@ export async function syncGit(): Promise<void> {
 }
 
 // istanbul ignore next
-export async function getRepoStatus(): Promise<StatusResult> {
+export async function getRepoStatus(
+  options?: TaskOptions
+): Promise<StatusResult> {
```

Comment:
We should only allow a single string, which must be a relative path in repo.

Reply:
Would you like it to be changed to e.g. `path?: string` instead of a `TaskOptions`?

Reply:
Hey @viceice, Questions:

Is this because you don't want other `git status` options other than the file path to be specified via this function? 

Also, how would you suggest to ensure this part:`must be a relative path in repo`? 
would joining the `localDir` from globalConfig do the job?

Reply:
yes, something like in
- #16125